generators/generate-blueprint/templates/cli/cli.mjs.ejs

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+import { runJHipster, done, logger } from 'generator-jhipster/esm/cli';
+import { readFile } from 'fs/promises';
+import { fileURLToPath } from 'url';
+import { dirname, basename } from 'path';
+
+// Get package name to use as namespace.
+// Allows blueprints to be aliased.
+const packagePath = dirname(dirname(fileURLToPath(import.meta.url)));
+const packageFolderName = basename(packagePath);
+
+(async () => {
```

Comment:
Not a fan of providing CLI for a blueprint. But its up to you

Reply:
See the use case:
A blueprint requires a specific jhipster version so a generator-jhipster-foo requires the end user to install generator-jhipster@7.4.1, not 7.4.0 nor 7.5.0.

A custom cli will use the desired generator-jhipster without having to manually install it.